modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/OpcUaProtocolAdapter.java

```diff
@@ -342,15 +341,18 @@ private void createClient() throws UaException {
         opcUaClient.addSessionActivityListener(new SessionActivityListener() {
             @Override
             public void onSessionInactive(final @NotNull UaSession session) {
+                log.info("OpcUa client disconnected: {}", session);
```

Comment:
```suggestion
                log.info("OPC UA client disconnected: {}", session);
```

Reply:
In general: Maybe put the adapter-id into all log statements, to distinguish if there are multiple opc ua adapters.

Reply:
Good idea!